tests/foreman/cli/test_organization.py

```diff
@@ -903,24 +988,24 @@ def test_positive_add_capsule_by_name(self):
             'name': org['name'],
             'smart-proxy': proxy['name'],
         })
+        org = Org.info({'name': org['name']})
+        self.assertIn(proxy['name'], org['smart-proxies'])
 
     @run_only_on('sat')
     @tier2
     @stubbed()
     def test_positive_add_capsule_by_id(self):
-        """@test: Add a capsule by its ID
+        """Add a capsule to organization by its ID
 
         @feature: Organization
 
         @assert: Capsule is added to the org
-
-        @status: manual
         """
 
     @run_only_on('sat')
     @stubbed("Needs to be re-worked!")
```

Comment:
Please check the stubbed docstring & code. All it does is basically this:
```python
@unittest2.skip(reason)
@pytest.mark.stubbed
def func(...):
    ....
```
So that's the same as using unittest.skip + marking the test as stubbed. What's the point of using unittest.skip only?